baselines/common/mpi_moments.py

```diff
@@ -9,7 +9,7 @@ def mpi_mean(x, axis=0, comm=None, keepdims=False):
     if comm is None: comm = MPI.COMM_WORLD
     xsum = x.sum(axis=axis, keepdims=keepdims)
     n = xsum.size
-    localsum = np.zeros(n+1, x.dtype)
+    localsum = np.zeros(n+1, np.float32) # x.dtype)
```

Comment:
is this change necessary? If I understand the logic correctly, you'll already have a correct dtype at this point (at least when being called from `her.mpi_average`)